### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by joshcanhelp**
- Fix code sample in the docs
- Link directly to configuration
- Fix eslint picking up on coverage

### CODE REVIEW

**Review by stevehobbsdev — APPROVED** [review]
🎉

**Comment by joshcanhelp** [comment]
Just FYI, look like there is a flaky test for "logout route > should perform a distributed logout" ... docs change caused a fail and re-running fixed it.

File: index.d.ts [hunk 1]
```diff
@@ -197,13 +197,14 @@ interface LogoutOptions {
 /**
  * Configuration parameters passed to the `auth()` middleware.
  *
- * {@link ConfigParams.issuerBaseURL issuerBaseURL}, {@link ConfigParams.baseURL baseURL}, {@link ConfigParams.clientID clientID}
+ * {@link ConfigParams.issuerBaseURL issuerBaseURL}, {@link ConfigParams.baseURL baseURL}, {@link ConfigParams.clientID clientID}, {@link ConfigParams.clientSecret clientSecret}
```
Comment (adamjmcgrath) [inline index.d.ts:200]: `clientSecret` isn't required by default, only if you're doing code flow

**Review by adamjmcgrath — CHANGES_REQUESTED** [review]

File: index.d.ts (hunk 1 above)
Comment (joshcanhelp) [reply index.d.ts:200]: True 👍 but it can be included with env vars so I'll re-word a bit.

**Review by adamjmcgrath — APPROVED** [review]

**Comment by adamjmcgrath** [comment]
Thanks @joshcanhelp!

**Pull request merged**